### ISSUE TITLE
Нема відступів по бокам

### ISSUE THREAD

**Issue by bucefal91** [Andrukas8/responsive_design_practice#63]
Можливо це пов"язано з попереднім тікетом про горизонтальний скролл, але на перших 2х блоках після хедеру на певній ширині нема горизонтальних відступів і воно виглядає дуже погано.
![screenshot from 2017-07-11 13-17-29](https://user-images.githubusercontent.com/17630965/28083615-8bf81d10-663b-11e7-9fc3-f6bec12faa1d.png)

**Comment by bucefal91** [comment]
До речі, знизу те саме спостерігається - багато блоків не мають горизонтальних маржинів.

**Comment by Andrukas8** [comment]
тут знову ж таки в моїх браузерах з відступами наче все добре

![image](https://user-images.githubusercontent.com/21954163/28199629-c52cab9c-686f-11e7-9f3b-c4291b1c9482.png)

**Comment by bucefal91** [comment]
Спробуй у firefox. Відкрий javascript console у debugging tools, і впиши туди команду: `window.innerWidth` у відповідь воно тобі скаже поточну ширину вебсторінки.

В мене бага спостерігається, на приклад, на 990 пікселях. Спробуй у себе цю ширину, будь ласка.

**Comment by Andrukas8** [comment]
так, побачив. зробив, не знаю чи правильний підхід обрав, але наче працює.
